draft-ietf-mmusic-sdp-bundle-negotiation.xml

```diff
@@ -1119,21 +1114,115 @@ SDP Answer
           from SSRC to identification-tag is received. Note that the mapping
           from SSRC to identification-tag can change at any time during an RTP
           session.
+	</t>
+
+
+      <t>Note: The following algorithm does not yet have WG consensus
+      but is included here as something concrete for the working group
+      to discuss.</t>

```

Comment:
Please remove this statement. It does not belong even in proposals for text in drafts in post-WG-last-call state. (this and some of the other comments in this review come from https://github.com/rtcweb-wg/jsep/issues/431)